app-tasks/dags/aoi-monitoring/update_aoi_projects.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import logging
+import os
+
+from airflow.models import DAG
+from airflow.operators import PythonOperator, BranchPythonOperator
+
+from rf.utils.exception_reporting import wrap_rollbar
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 4, 1)
+}
+
+dag = DAG(
+    dag_id='update_aoi_projects',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=os.getenv('AIRFLOW_DAG_CONCURRENCY', 24)
+)
+
+
+def check_whether_update_available(**context):
+    # check whether a project needs to update
```

Comment:
Yeah, I see what you mean -- though seems like it adds a little complexity too this way -- the query for determining if there are scenes to add is going to be almost the same as adding them. I'd prefer less nodes for now rather than more. 

Also if this is going to be run as `java -jar` we pay the startup penalty twice. Either way, both of our arguments are kind of conjecture at this point without much data so maybe consolidate them for now and we can split them later if problems are introduced by combining them.